### PR TITLE
Clietn model

### PR DESCRIPTION
**Pull request by HMTanzeel**

### CODE REVIEW

File: db/migrate/20220322131347_create_clients.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class CreateClients < ActiveRecord::Migration[7.0]
+  def change
+    create_table :clients do |t|
+      t.string :first_name
```
Comment (aliammaar) [inline db/migrate/20220322131347_create_clients.rb:4]: add null false for first_name, last_name and email

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  it "valid if creates a user" do
+    expect(create(:user)).to be_valid
+  end
+
+  it "valid if name is available" do
+    expect(create(:user).name).to be_present
+  end
+
+  it "valid if id is matched" do
```
Comment (aliammaar) [inline spec/models/user_spec.rb]: vague spec, I don't understand what you are testing here.

File: spec/models/project_spec.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  it { should have_db_column(:title).of_type(:string) }
+  it { should have_db_column(:description).of_type(:string) }
+  it { should have_db_column(:active).of_type(:boolean) }
+  it { should have_db_column(:project_type).of_type(:integer) }
+  it { should have_db_column(:user_id).of_type(:integer) }
+  it { should have_db_column(:started_at).of_type(:datetime) }
+  it { should have_db_column(:close_at).of_type(:datetime) }
+  it { should belong_to(:user) }
+
+  it "valid if title is available" do
```
Comment (aliammaar) [inline spec/models/project_spec.rb:13]: What you are trying to test here ?

Comment (HMTanzeel) [reply spec/models/project_spec.rb:13]: on 'title' has a validation of null false.

Comment (aliammaar) [reply spec/models/project_spec.rb:13]: remove this.

File: spec/models/user_spec.rb (hunk 2 above)
Comment (aliammaar) [reply spec/models/user_spec.rb]: remove this as well.

**Pull request closed without merging**